storage/ci/phinx.php.example

```diff
@@ -11,7 +11,7 @@ return [
         'production'              => [
             'adapter' => 'mysql',
             'host'    => getenv('DATA_API_MYSQL_HOST'),
-            'name'    => 'gonano',
+            'name'    => getenv('DATA_API_MYSQL_NAME'),
             'user'    => getenv('DATA_API_MYSQL_USER'),
             'pass'    => getenv('DATA_API_MYSQL_PASS'),
             'port'    => 3306,
```

Comment:
Can you wrap this to the env var?

Reply:
Will refactor current approach to generic and more simple, as discussed before.